bundle/manifests/metallb-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,382 @@
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                resources: {}
+              - args:
+                - --metrics-addr=127.0.0.1:8080
+                - --enable-leader-election
+                command:
+                - /manager
+                env:
+                - name: SPEAKER_IMAGE
+                  value: quay.io/metallb/speaker:main
+                - name: CONTROLLER_IMAGE
+                  value: quay.io/metallb/controller:main
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                image: quay.io/metallb/metallb-operator:latest
+                name: manager
+                resources:
+                  limits:
+                    cpu: 100m
+                    memory: 30Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+              terminationGracePeriodSeconds: 10
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+        - apiGroups:
+          - ""
+          resourceNames:
+          - memberlist
+          resources:
+          - secrets
+          verbs:
+          - list
+        - apiGroups:
+          - apps
+          resourceNames:
+          - controller
+          resources:
+          - deployments
+          verbs:
+          - get
+        serviceAccountName: controller
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        - apiGroups:
+          - apps
+          resources:
+          - daemonsets
+          - deployments
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: default
```

Comment:
lets not use the default one for the operator lets give him a service account

Reply:
@schseba talked about this with @mmirecki @fedepaol and we saw that other operators don't necessarily change that.
If we do decide to give it a service account I think it'll be better to do it in a different PR because it will also affect the regular deployment (without olm)

Reply:
No problem lets have this one in a following up PR.
and you are right we should have another SA yaml in the regular conf folder to support this